client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -105,18 +110,14 @@ class ActivityLogItem extends Component {
 	}
 
 	getSummary() {
-		const {
-			translate,
-			actionText
-		} = this.props;
+		const { translate } = this.props;
 
-		return ( actionText &&
+		return (
 			<div className="activity-log-item__action">
 				<EllipsisMenu position="bottom right">
-					<PopoverMenuItem onClick={ noop } icon="undo">{ actionText }</PopoverMenuItem>
-					<PopoverMenuItem icon="pencil">Option B</PopoverMenuItem>
-					<PopoverMenuSeparator />
-					<PopoverMenuItem icon="help">{ translate( 'More Info' ) }</PopoverMenuItem>
+					<PopoverMenuItem onClick={ this.handleClickRestore } icon="undo">
```

Comment:
This came from _the big PR_.

@keoshi - Do we use the same `history` icon for activity item rewinds, or another icon?